ballerina/database.bal

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java;
+
+# Represents a MongoDB database.
+@display {
+    label: "MongoDB Database"
+}
+public isolated client class Database {
+
+    private final Client 'client;
+
+    isolated function init(Client 'client, string databaseName) returns Error? {
+        self.'client = 'client;
```

Comment:
What is the usage of `client` instance other than in the init-phase ?

Reply:
No need. Will update the code.